--- matchbox-panel-0.9.3/applets/Makefile.am~	2005-10-03 00:06:58.000000000 +0200
+++ matchbox-panel-0.9.3/applets/Makefile.am	2015-11-22 00:47:12.783853175 +0100
@@ -38,7 +38,7 @@
 mb_applet_clock_SOURCES =          mb-applet-clock.c
 mb_applet_menu_launcher_SOURCES =  mb-applet-menu-launcher.c
 mb_applet_wireless_SOURCES =       mb-applet-wireless.c
-mb_applet_wireless_LDADD   =       $(LIBMB_LIBS) $(SN_LIBS) $(WIRELESS_LIBS)
+mb_applet_wireless_LDADD   =       $(LIBMB_LIBS) $(SN_LIBS) $(WIRELESS_LIBS) -lm
 mb_applet_launcher_SOURCES =       mb-applet-launcher.c  
 mb_applet_system_monitor_SOURCES = mb-applet-system-monitor.c 
 
